frontend/src/v5/ui/routes/dashboard/projects/tickets/ticketsTable/ticketsTable.component.tsx

```diff
@@ -57,32 +59,27 @@ export const TicketsTable = () => {
 	const history = useHistory();
 	const params = useParams<DashboardTicketsParams>();
 	const { teamspace, project, template, ticketId } = params;
-	const prevTemplate = useRef(undefined);
+	const { groupBy, fetchColumn } = useContext(TicketsTableContext);
```

Comment:
When the user is grouping by a property that is still being fetch, the ticket that is being loaded gets shoved into 'unset' . 
Is it too hard to hide them in the meanwhile?  Ideally we should have an indication that we are still loading data. (talk to @apeitup  about it)  

<img width="671" alt="image" src="https://github.com/user-attachments/assets/96742ceb-865c-4c06-adef-7305595aa4db" />